bluebottle/assignments/tests/test_models.py

```diff
@@ -55,6 +55,18 @@ def test_date_changed(self):
             else:
                 self.assertFalse(participant.user.email in recipients)
 
+    def test_end_date_type_changed(self):
+        assignment = AssignmentFactory(
+            title='Test Title',
+            status='open',
+            end_date_type='on_date',
+            preparation=5,
+            date=now() + timedelta(days=4),
+        )
+
+        assignment.end_date_type = 'deadline'
+        assignment.save()
+
```

Comment:
This should assert at least something. Without is it just tests that no exceptions are thrown.

Reply:
assertions added